src/main/java/com/automationexercise/Utilities/Utility.java

```diff
@@ -199,5 +200,44 @@ public static void openWebsite(String URL) {
         getDriver().get(URL);
         implicitlyWait();
         LogUtils.info("Open website with URL: " + URL);
+    }
+        /** Get element location
+         * @param Locator the specified element
+          */
+
+    public static boolean getElementLocation(By Locator)
+    {
+        boolean isAtTop = (boolean) ((JavascriptExecutor) getDriver()).executeScript(
+                "return arguments[0].getBoundingClientRect().top <= 0", Locator);
+
+        // Output the result
+        if (isAtTop) {
+            System.out.println("Element is at the top of the page.");
+            return true;
+        } else {
+            System.out.println("Element is not at the top of the page.");
+            return false;
+        }
+    }
+
+    public static void hoverElement(WebElement Locator)
+    {
+     //   WebElement element = getDriver().findElement(Locator);
+
+        // Create an instance of Actions class
+        Actions actions = new Actions(getDriver());
+
+        // Perform hover over the element
+        actions.moveToElement(Locator).perform();
+        actions.click().perform();
+    }
+    /**
+     * Scroll to specific element using it's locator
+     * @param locator
+     */
+    public static void ScrollToElement (WebElement locator) {
```

Comment:
There are a 3 functions for scrolling exist on Utility you can use them and deleted this duplicated function: 
```

    public static void scrollToElementAtTop(WebDriver driver, By locator) {
        ((JavascriptExecutor) (driver)).executeScript("arguments[0].scrollIntoView(true);", findWebElement(driver, locator));
    }

    public static void scrollToElementAtBottom(WebDriver driver, By locator) {
        ((JavascriptExecutor) (driver)).executeScript("arguments[0].scrollIntoView(false);", findWebElement(driver, locator));
    }

    //TODO:  Scroll to specific position
    public static void scrollToPosition(WebDriver driver, int x, int y) {
        ((JavascriptExecutor) (driver)).executeScript("window.scrollTo(" + x + ", " + y + ");");
    }
```